### ISSUE TITLE
import { tailwindToCSS } from "tw-to-css"; Named export 'tailwindToCSS' not found. 

### ISSUE THREAD

**Issue by faradaytrs** [resendlabs/react-email#1002]
### Describe the Bug

Error in runtime when trying to run.
````
node build/server.js --dns-result-order=ipv4first
file:///app/node_modules/.pnpm/@react-email+tailwind@0.0.11_react@18.2.0/node_modules/@react-email/tailwind/dist/index.mjs:24
import { tailwindToCSS } from "tw-to-css";
         ^^^^^^^^^^^^^
SyntaxError: Named export 'tailwindToCSS' not found. The requested module 'tw-to-css' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'tw-to-css';
const { tailwindToCSS } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:131:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:213:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.9.0
  ELIFECYCLE  Command failed with exit code 1.
````

### Which package is affected (leave empty if unsure)

@react-email/tailwind

### Link to the code that reproduces this issue

-

### To Reproduce

use tailwind and run project with node-js 

### Expected Behavior

no error

### What's your node version? (if relevant)

20.9.0

**Comment by faradaytrs** [comment]
if you don't know what is the problem i can try to setup a project to reproduce.
We are using esm.

**Comment by lunoob** [comment]
<img width="1336" alt="image" src="https://github.com/resendlabs/react-email/assets/49156607/33d09c75-76e9-4c43-b960-b52d169b7ab1">
This may be caused by tw-to-css

**Comment by almeraikhi** [comment]
I'm facing the same issue, though it only happens when I import the email component into my own NextJs app.
Running react-email dev environment doesn't seem to be causing this issue for me.

**Comment by JClackett** [comment]
Since updating to `"@react-email/components": "0.0.9"`, my Remix app won't build due to some import issues:

```
✘ [ERROR] Could not resolve "react-property"

    ../../node_modules/@react-email/components/node_modules/html-react-parser/lib/attributes-to-props.js:1:28:
      1 │ var reactProperty = require('react-property');
        ╵                             ~~~~~~~~~~~~~~~~

  You can mark the path "react-property" as external to exclude it from the bundle, which will remove this error. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.


✘ [ERROR] Could not resolve "style-to-js"

    ../../node_modules/@react-email/components/node_modules/html-react-parser/lib/utilities.js:2:24:
      2 │ var styleToJS = require('style-to-js').default;
        ╵                         ~~~~~~~~~~~~~

  You can mark the path "style-to-js" as external to exclude it from the bundle, which will remove this error. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.
```

0.0.8 continues to work still

**Comment by almeraikhi** [comment]
> 0.0.7 continues to work still

since I installed `@react-email/tailwind` individually, I downgraded it to `0.0.8`, and it works now! :)

**Comment by bukinoshita** [comment]
Does it work if you install the `tw-to-css` package?

**Comment by almeraikhi** [comment]
> Does it work if you install the `tw-to-css` package?

I tried that, and it didn't work.

For some reason `@react-email/tailwind` "assumes" that it's importing the CommonJS version of `tw-to-css` even though it's importing the ESM one.

**Comment by bukinoshita** [comment]
Just released a `@react-email/component@0.0.10-canary.0` that fixes that issue. Will be releasing a non-canary version soon
